modes/lms/evil-collection-lms.el

```diff
@@ -0,0 +1,209 @@
+
+(declare-function lms-ui-playing-now-refresh "lms")
+
+(defconst evil-collection-lms-maps '(lms-ui-players-mode-map
+                                     lms-ui-playing-now-mode-map
+                                     lms-ui-playlist-mode-map
+                                     lms-ui-track-info-mode-map
+                                     lms-ui-tracks-list-mode-map
+                                     lms-ui-year-album-artist-list-mode-map))
+
+(defconst evil-collection-lms-modes '(lms-players-mode
+                                      lms-ui-playing-now-mode
+                                      lms-ui-track-info-mode
+                                      lms-ui-players-mode
+                                      lms-ui-playlist-mode
+                                      lms-ui-tracks-list-mode
+                                      lms-ui-year-album-artist-list-mode))
+
+(defun evil-collection-lms-ui-track-info-mode-quit ()
+  "Exit lms-ui-track-info-mode."
+  (interactive)
+  (kill-buffer "*LMS: Track Information*")
+  (lms-ui-playing-now-refresh))
+
+(defun evil-collection-lms-ui-players-mode-quit ()
+  "Exit lms-ui-players-mode."
+  (interactive)
+  (kill-buffer (format "*LMS: Players*"))
+  (lms-ui-playing-now-refresh))
+
+(defun evil-collection-lms-ui-playlist-mode-quit ()
+  "Exit lms-ui-playlist-mode."
+  (interactive)
+  (kill-buffer (format "*LMS: Playlist [%d tracks]*" (length lms--ui-pl-tracks)))
+  (lms-ui-playing-now-refresh))
+
+(defun evil-collection-lms-ui-tracks-list-mode-quit ()
+  "Exit lms-ui-tracks-list-mode."
+  (interactive)
+  (kill-buffer)
+  (lms-ui-playing-now-refresh))
+
+(defalias 'evil-collection-lms-ui-year-album-artist-list-mode-quit
+  'evil-collection-lms-ui-tracks-list-mode-quit
+  "Exit lms-ui-tracks-list-mode.")
+
+(defun evil-collection-lms-goto-bottom ()
+  "Move the point to the last penultimate line of the current buffer."
+  (goto-char (max-char))
+  (forward-line -1))
+
+(defun evil-collection-lms-next-line ()
+  "Move the point to the next line, except if it is at the end of the buffer."
+  (interactive)
+  (forward-line 1)
+  (when (eobp) (forward-line -1)))
+
+
+;;;###autoload
+(defun evil-collection-lms-setup ()
+  "Set up `evil' bindings for `lms'."
+
+  (dolist (map evil-collection-lms-maps)
+    (evil-collection-inhibit-insert-state map))
+
+  (dolist (mode evil-collection-lms-modes)
+    (evil-set-initial-state mode 'normal))
+
+  (evil-collection-define-key 'normal 'lms-ui-playing-now-mode-map
+    "t"              'lms-ui-playing-now-change-player-power-state
+    (kbd "C-p")      'lms-ui-playing-now-players-list
+    "R"              'lms-ui-playing-now-change-rating
+    (kbd  "C-=")     'lms-ui-playing-now-set-volume
+    "x"              'lms-ui-playing-now-play-pause
+    (kbd "RET")      'lms-ui-playing-now-play-pause
+    "p"              'lms-ui-playing-now-play
+    "s"              'lms-ui-playing-now-stop
+    "J"              'lms-ui-playing-now-next
```

Comment:
I personally prefer <kbd>C-j</kbd> over <kbd>J</kbd>. <kbd>J</kbd> is used to jump, <kbd>K</kbd> is for lookup.